ssv/committee.go

```diff
@@ -0,0 +1,187 @@
+package ssv
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+
+	spec "github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/pkg/errors"
+	"github.com/ssvlabs/ssv-spec/qbft"
+	"github.com/ssvlabs/ssv-spec/types"
+)
+
+type CreateRunnerFn func(shareMap map[spec.ValidatorIndex]*types.Share) *CommitteeRunner
+
+type Committee struct {
+	Runners           map[spec.Slot]*CommitteeRunner
```

Comment:
that's an implementation detail imo

Spec creates a runner for each slot.
Implementation can deal with memory issues.
I believe impl has this limitation because this was discussed with Moshe a long time ago

Reply:
right now we are not limiting runners in implementaion but we are planning to do something like 32 slots and remove old ones as new are added.